eclog/src/main/java/com/ericsson/bss/cassandra/ecaudit/eclog/LogPrinter.java

```diff
@@ -48,20 +47,20 @@ void print(QueueReader queueReader)
         {
             while (isEligibleForPrint(queueReader, printedRecords))
             {
-                AuditRecord auditEntry = queueReader.nextRecord();
+                StoredAuditRecord auditEntry = queueReader.nextRecord();
 
                 StringBuilder builder = new StringBuilder();
-                builder.append(auditEntry.getTimestamp()).append('|');
-                builder.append(auditEntry.getClientAddress().getAddress().getHostAddress())
-                       .append(portIfPresent(auditEntry.getClientAddress()))
-                       .append('|');
-                builder.append(auditEntry.getCoordinatorAddress().getHostAddress()).append('|');
-                builder.append(auditEntry.getUser()).append('|');
-                builder.append(auditEntry.getStatus()).append('|');
-                auditEntry.getBatchId()
-                          .ifPresent(bid -> builder.append(bid).append('|'));
-
-                builder.append(auditEntry.getOperation().getOperationString());
+
+                auditEntry.getTimestamp().ifPresent(timestamp -> builder.append(timestamp).append('|'));
+                builder.append(getClient(auditEntry));
```

Comment:
That is true, this is an issue since fields/parameters in the ecLog output is not "tagged" with its type (which is done when logging which SLF4J)... But I think I will leave it like this for now... 
This will be fixed in issue #102, which is a continuation of this issue.

Reply:
I added a comment to #102 regarding this